contracts/CarbonVoteX.sol

```diff
@@ -101,9 +124,17 @@ contract CarbonVoteX {
     // @param endBlock ending block (unix timestamp) of the event
     // @param pollId UUID (hash value) of a poll
     // @param tokenAddr the address of the token
+    // a restricted function
     // register a new poll
     // Note that we do not allow re-registrations for the same UUID
-    function register(uint startBlock, uint endBlock, bytes32 pollId, address tokenAddr) public {
+    function register(
```

Comment:
see Function Declaration in http://solidity.readthedocs.io/en/v0.4.24/style-guide.html

```{solidity}
function register(
    uint startBlock,    
    uint endBlock, 
    bytes32 pollId, 
    address tokenAddr
) 
    public
    auth
{
```